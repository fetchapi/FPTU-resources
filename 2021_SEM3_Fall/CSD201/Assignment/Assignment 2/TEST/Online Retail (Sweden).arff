version https://git-lfs.github.com/spec/v1
oid sha256:d7e7d7834df7654f5e3234fbd4db6bc119edfd46c0fae30d137f0e2554647a93
size 32343
