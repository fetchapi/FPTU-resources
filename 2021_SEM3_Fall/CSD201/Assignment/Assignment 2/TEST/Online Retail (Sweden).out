84997D:3
POST 84997D:3
23508:3
23494:3
23310:3
23236 23310:3
23241 23310:3
23241 23236 23310:3
23242 23310:3
23242 23236 23310:3
23241 23242 23310:3
23241 23242 23236 23310:3
23307:3
22961 23307:3
22720 23307:3
22961 22720 23307:3
23306:3
22961 23306:3
23298:3
POST 23298:3
23297:3
23293 23297:3
23296:3
23293 23296:3
23295 23296:3
23293 23295 23296:3
23295:3
23293 23295:3
23245:3
23242:3
23236 23242:3
23241 23242:3
23241 23236 23242:3
23241:3
23236 23241:3
23235:3
23191:3
22961 23191:3
22938:3
22909:3
22952 22909:3
22721:3
22489 22721:3
22668:3
POST 22668:3
22536:3
22530 22536:3
22720 22536:3
22530 22720 22536:3
85178 22536:3
85178 22530 22536:3
85178 22720 22536:3
85178 22530 22720 22536:3
22530:3
22720 22530:3
85178 22530:3
85178 22720 22530:3
22489:3
22348:3
22326:3
22328 22326:3
22090:3
POST 22090:3
21791:3
22720 21791:3
22492 21791:3
22492 22720 21791:3
20724:3
POST 20724:3
22378 20724:3
POST 22378 20724:3
15036:3
85178:4
22720 85178:3
23480:4
23084 23480:3
POST 23480:3
23084 POST 23480:3
23293:4
23236 23293:3
23236:4
84077 23236:3
23084:4
21731 23084:3
POST 23084:4
21731 POST 23084:3
22952:4
22951:4
22417 22951:3
21212 22951:3
22417 21212 22951:3
22492 22951:3
21210 22951:3
22720 22951:3
22417 22720 22951:3
21212 22720 22951:3
22417 21212 22720 22951:3
22417:4
21212 22417:4
22720 22417:3
21212 22720 22417:3
22328:4
POST 22328:3
22139:4
22720 22139:3
21977:4
21889:4
22492 21889:3
21212:4
22720 21212:3
21210:4
84077:5
22961 84077:3
22961:5
22492 22961:3
22720 22961:3
22467:5
POST 22467:5
22378:5
21731 22378:4
POST 22378:5
21731 POST 22378:4
21731:5
POST 21731:5
22779:6
POST 22779:3
22492:6
22720 22492:3
22720:7
POST:24
